docs/resources/kernel_module.md.erb

```diff
@@ -10,14 +10,49 @@ Use the `kernel_module` InSpec audit resource to test kernel modules on Linux pl
 
 A `kernel_module` resource block declares a module name, and then tests if that module is a loadable kernel module:
 
-    describe kernel_module('module_name') do
-      it { should be_loaded }
-    end
+  describe kernel_module('video') do
```

Comment:
I think we should keep the generic part here and move the examples into the examples sections to keep consistent with other docs